spec/spec_helper.rb

```diff
@@ -8,6 +8,26 @@
 ])
 SimpleCov.start
 
+require_relative './reset_test_database.rb'
```

Comment:
in the review doc they say to ensure asset routes are set correctly. I'm sure you've done this elsewhere, but I haven't been able to find it. 

https://github.com/emilyalice2708/chitter-challenge/blob/master/docs/review.md#ensure-asset-routes-are-set-correctly